src/Units/BomberModel.java

```diff
@@ -1,17 +1,24 @@
 package Units;
 
 import Engine.GameAction.ActionType;
+import Units.MoveTypes.Flight;
+import Units.MoveTypes.MoveType;
 import Units.Weapons.BomberBombs;
 import Units.Weapons.WeaponModel;
 
-public class BomberModel extends AirModel
+public class BomberModel extends UnitModel
 {
+  private static final int UNIT_COST = 2200;
```

Comment:
I suppose I ought to get around to changing these to the DoR numbers?
Maybe when/if we add fog vision radius?